### PR TITLE
feat: Added Alert and Button components

### PR DESCRIPTION
**Pull request by arjunbharti**
Added two new components:

    1. Alert

    2. Button
        a. Standard Buttons
        b. Link Buttons
        c. Icon Buttons 
        d. Floating Action Buttons

### CODE REVIEW

**Review by bhavyaj12 — APPROVED** [review]
Looks good to go, alert colours are nice and clean

File: components/alert.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    min-width: 50%;
+}
+
+.alert-icon{
+    margin-right: 1rem;
+    font-size: 1.5rem;
+}
+
+.alert-success{
+    background-color: var(--light-green);
```
Comment (SHIKHASINGH1506) [inline components/alert.css]: it's better to name color variables as the purpose instead of the color itself. For example "success-color"

**Review by SHIKHASINGH1506 — CHANGES_REQUESTED** [review]
left a comment rest look good

**Comment by ark2002** [comment]
hey , i think the sidebar isnt working like after clicking on any of the components from the list it just brings me to the top of the page

**Comment by arjunbharti** [comment]
> hey , i think the sidebar isnt working like after clicking on any of the components from the list it just brings me to the top of the page

Yup, haven't worked on it yet. Will do it in the end. Please ignore that for now

File: components/alert.css (hunk 1 above)
Comment (arjunbharti) [reply components/alert.css]: Sure, will make the changes

Comment (arjunbharti) [reply components/alert.css]: Made the changes. Thanks!

**Pull request merged**